aws/registers/registers.tf

```diff
@@ -0,0 +1,448 @@
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "school-phase_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "school-phase", false)}"
+
+  name = "school-phase"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "school-tag_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "school-tag", false)}"
+
+  name = "school-tag"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "school-trust_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "school-trust", false)}"
+
+  name = "school-trust"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "school-type_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "school-type", false)}"
+
+  name = "school-type"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "street_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "street", false)}"
+
+  name = "street"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "street-custodian_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "street-custodian", false)}"
+
+  name = "street-custodian"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "territory_register" {
+  source = "../modules/register"
+  enabled = "${lookup(var.enabled_registers, "territory", false)}"
+
+  name = "territory"
+  environment = "${var.vpc_name}"
+  load_balancer = "${module.multi.load_balancer}"
+  dns_zone_id = "${aws_route53_zone.temporary_hosted_zone.zone_id}"
+
+  enable_availability_checks = "${var.enable_availability_checks}"
+}
+
+module "uk_register" {
```

Comment:
this will need an update to the README (probably after #246 is merged) to document an extra step for creating a register